includes/wikia/traits/PowerUserTrait.php

```diff
@@ -0,0 +1,40 @@
+<?php
+/**
+ * A trait adding PowerUser related methods.
+ * Intended to be used by the User class.
+ *
+ * @package PowerUser
+ * @author Adam Karmiński <adamk@wikia-inc.com>
+ */
+use Wikia\PowerUser\PowerUser;
```

Comment:
Than it makes sense to load the setup file in `includes/wikia/DefaultSettings.php`.